packages/cf-deploy-config-writer/test/unit/__snapshots__/cap.test.ts.snap

```diff
@@ -277,16 +277,16 @@ builder:
   },
   "package.json": Object {
     "contents": "{
-    \\"cds\\": {},
```

Comment:
Nothng has changed, just formatting changes since its now using a different writer method to update the package.json.